<ol>
<li><div id="Knowles2017dox"><em>Knowles* DA, Burrows* CK, Blischak JD, Patterson KM, Ober C, Pritchard JK and Gilad Y</em> (2017), <strong>"Determining the genetic basis of anthracycline-cardiotoxicity by molecular response QTL mapping in induced cardiomyocytes"</strong>, bioRxiv.  *These authors contributed equally to this work.</div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2017dox">Abstract</a>][<a href="#bib_Knowles2017dox" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/212381" target="_blank">DOI</a>]</div>

<div id="abs_Knowles2017dox" class="collapse">
	<b>Abstract</b>: Anthracycline-induced cardiotoxicity (ACT) is a key limiting factor in setting optimal chemotherapy regimes for cancer patients, with almost half of patients expected to ultimately develop congestive heart failure given high drug doses. However, the genetic basis of sensitivity to anthracyclines such as doxorubicin remains unclear. To begin addressing this, we created a panel of iPSC-derived cardiomyocytes from 45 individuals and performed RNA-seq after 24h exposure to varying levels of doxorubicin. The transcriptomic response to doxorubicin is substantial, with the majority of genes being differentially expressed across treatments of different concentrations and over 6000 genes showing evidence of differential splicing. Overall, our observations indicate that that splicing fidelity decreases in the presence of doxorubicin. We detect 376 response-expression QTLs and 42 response-splicing QTLs, i.e. genetic variants that modulate the individual transcriptomic response to doxorubicin in terms of expression and splicing changes respectively. We show that inter-individual variation in transcriptional response is predictive of cell damage measured <em>in vitro</em> using a cardiac troponin assay, which in turn is shown to be associated with <em>in vivo</em> ACT risk. Finally, the molecular QTLs we detected are enriched in lower ACT GWAS p-values, further supporting the <em>in vivo</em> relevance of our map of genetic regulation of cellular response to anthracyclines.
</div>
<div id="bib_Knowles2017dox" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2017dox,
  author = {Knowles*, David A and Burrows*, Courtney K and Blischak, John D and Patterson, Kristen M and Ober, Carole and Pritchard, Jonathan K and Gilad, Yoav},
  title = {Determining the genetic basis of anthracycline-cardiotoxicity by molecular response QTL mapping in induced cardiomyocytes},
  journal = {bioRxiv},
  year = {2017},
  doi = {10.1101/212381}
}
</pre>
</div>
</li><li><div id="Wainberg206961"><em>Wainberg M, Sinnott-Armstrong N, Knowles DA, Golan D, Ermel R, Ruusalepp A, Quertermous T, Hao K, Bjorkegren JLM, Rivas MA and Kundaje A</em> (2017), <strong>"Vulnerabilities of transcriptome-wide association studies"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Wainberg206961">Abstract</a>][<a href="#bib_Wainberg206961" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/206961" target="_blank">DOI</a>] [<a href="https://www.biorxiv.org/content/early/2017/10/27/206961" target="_blank">URL</a>]</div>

<div id="abs_Wainberg206961" class="collapse">
	<b>Abstract</b>: Transcriptome-wide association studies (TWAS) integrate GWAS and expression quantitative trait locus (eQTL) datasets to discover candidate causal gene-trait associations. We integrate multi-tissue expression panels and summary GWAS for LDL cholesterol and Crohn's disease to show that TWAS are highly vulnerable to discovering non-causal genes, because variants at a single GWAS hit locus are often eQTLs for multiple genes. TWAS exhibit acute instability when the tissue of the expression panel is changed: candidate causal genes that are TWAS hits in one tissue are usually no longer hits in another, due to lack of expression or strong eQTLs, while non-causal genes at the same loci remain. While TWAS is statistically valid when used as a weighted burden test to identify trait-associated loci, it is invalid to interpret TWAS associations as causal genes because the false discovery rate for TWAS causal gene discovery is not only high, but unquantifiable. More broadly, our results showcase limitations of using expression variation across individuals to determine causal genes at GWAS loci.
</div>
<div id="bib_Wainberg206961" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Wainberg206961,
  author = {Wainberg, Michael and Sinnott-Armstrong, Nasa and Knowles, David A and Golan, David and Ermel, Raili and Ruusalepp, Arno and Quertermous, Thomas and Hao, Ke and Bjorkegren, Johan L. M. and Rivas, Manuel A. and Kundaje, Anshul},
  title = {Vulnerabilities of transcriptome-wide association studies},
  journal = {bioRxiv},
  publisher = {Cold Spring Harbor Laboratory},
  year = {2017},
  url = {https://www.biorxiv.org/content/early/2017/10/27/206961},
  doi = {10.1101/206961}
}
</pre>
</div>
</li><li><div id="Taylor2017"><em>Taylor DL, Knowles DA, Scott LJ, Ramirez AH, Casale FP, Wolford BN, Guan L, Varshney A, Albanus RD, Parker SC, Narisu N, Chines PS, Erdos MR, Welch RP, Kinnunen L, Saramies J, Sundvall J, Lakka TA, Laakso M, Tuomilehto J, Koistinen HA, Stegle O, Boehnke M, Birney E and Collins FS</em> (2017), <strong>"Interactions between genetic variation and cellular environment in skeletal muscle gene expression"</strong>, bioRxiv: 10.1101/105429. </div>
<div>[<a data-toggle="collapse" href="#abs_Taylor2017">Abstract</a>][<a href="#bib_Taylor2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/105429" target="_blank">DOI</a>]</div>

<div id="abs_Taylor2017" class="collapse">
	<b>Abstract</b>: From whole organisms to individual cells, responses to environmental conditions are influenced by genetic makeup, where the effect of genetic variation on a trait depends on the environmental context. RNA-sequencing quantifies gene expression as a molecular trait, and is capable of capturing both genetic and environmental effects. In this study, we explore opportunities of using allele-specific expression (ASE) to discover cis acting genotype-environment interactions (GxE) - genetic effects on gene expression that depend on an environmental condition. Treating 17 common, clinical traits as approximations of the cellular environment of 267 skeletal muscle biopsies, we identify 10 candidate interaction quantitative trait loci (iQTLs) across 6 traits (12 unique gene-environment trait pairs; 10&percnt; FDR per trait) including sex, systolic blood pressure, and low-density lipoprotein cholesterol. Although using ASE is in principle a promising approach to detect GxE effects, replication of such signals can be challenging as validation requires harmonization of environmental traits across cohorts and a sufficient sampling of heterozygotes for a transcribed SNP. Comprehensive discovery and replication will require large human transcriptome datasets, or the integration of multiple transcribed SNPs, coupled with standardized clinical phenotyping.
</div>
<div id="bib_Taylor2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Taylor2017,
  author = {Taylor, D Leland and Knowles, David A and Scott, Laura J and Ramirez, Andrea H and Casale, Franceso Paolo and Wolford, Brooke N and Guan, Li and Varshney, Arushi and Albanus, Ricardo D'Oliveira and Parker, Stephen C.J. and Narisu Narisu and Chines, Peter S. and Erdos, Michael R. and Welch, Ryan P. and Kinnunen, Leena and Saramies, Jouko and Sundvall, Jouko and Lakka, Timo A. and Laakso, Markku and Tuomilehto, Jaakko and Koistinen, Heikki A. and Stegle, Oliver and Boehnke, Michael and Birney, Ewan and Collins, Francis S.},
  title = {Interactions between genetic variation and cellular environment in skeletal muscle gene expression},
  journal = {bioRxiv: 10.1101/105429},
  year = {2017},
  doi = {10.1101/105429}
}
</pre>
</div>
</li><li><div id="Knowles2015stochastic"><em>Knowles DA</em> (2015), <strong>"Stochastic gradient variational Bayes for gamma approximating distributions"</strong>, arXiv. , pp. 1509.01631.</div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2015stochastic">Abstract</a>][<a href="#bib_Knowles2015stochastic" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/1509.01631" target="_blank">URL</a>] [<a href="https://github.com/davidaknowles/gamma_sgvb" target="_blank">Code</a>]</div>

<div id="abs_Knowles2015stochastic" class="collapse">
	<b>Abstract</b>: While stochastic variational inference is relatively well known for scaling inference in Bayesian probabilistic models, related methods also offer ways to circumnavigate the approximation of analytically intractable expectations. The key challenge in either setting is controlling the variance of gradient estimates: recent work has shown that for continuous latent variables, particularly multivariate Gaussians, this can be achieved by using the gradient of the log posterior. In this paper we apply the same idea to gamma distributed latent variables given gamma variational distributions, enabling straightforward "black box" variational inference in models where sparsity and non-negativity are appropriate. We demonstrate the method on a recently proposed gamma process model for network data, as well as a novel sparse factor analysis. We outperform generic sampling algorithms and the approach of using Gaussian variational distributions on transformed variables.
</div>
<div id="bib_Knowles2015stochastic" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2015stochastic,
  author = {Knowles, David A},
  title = {Stochastic gradient variational Bayes for gamma approximating distributions},
  journal = {arXiv},
  year = {2015},
  pages = {1509.01631},
  url = {https://arxiv.org/abs/1509.01631}
}
</pre>
</div>
</li><li><div id="salimans2014using"><em>Salimans T and Knowles DA</em> (2014), <strong>"On using control variates with stochastic approximation for variational Bayes and its connection to stochastic linear regression"</strong>, arXiv. , pp. 1401.1022.</div>
<div>[<a data-toggle="collapse" href="#abs_salimans2014using">Abstract</a>][<a href="#bib_salimans2014using" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/1401.1022" target="_blank">URL</a>]</div>

<div id="abs_salimans2014using" class="collapse">
	<b>Abstract</b>: Recently, we and several other authors have written about the possibilities of using stochastic approximation techniques for fitting variational approximations to intractable Bayesian posterior distributions. Naive implementations of stochastic approximation suffer from high variance in this setting. Several authors have therefore suggested using control variates to reduce this variance, while we have taken a different but analogous approach to reducing the variance which we call stochastic linear regression. In this note we take the former perspective and derive the ideal set of control variates for stochastic approximation variational Bayes under a certain set of assumptions. We then show that using these control variates is closely related to using the stochastic linear regression approximation technique we proposed earlier. A simple example shows that our method for constructing control variates leads to stochastic estimators with much lower variance compared to other approaches.
</div>
<div id="bib_salimans2014using" class="collapse">
<b>BibTeX</b>:
<pre>
@article{salimans2014using,
  author = {Salimans, Tim and Knowles, David A},
  title = {On using control variates with stochastic approximation for variational Bayes and its connection to stochastic linear regression},
  journal = {arXiv},
  year = {2014},
  pages = {1401.1022},
  url = {https://arxiv.org/abs/1401.1022}
}
</pre>
</div>
</li><li><div id="palla2013dependent"><em>Palla* K, Knowles* DA and Ghahramani Z</em> (2013), <strong>"A dependent partition-valued process for multitask clustering and time evolving network modeling"</strong>, arXiv. , pp. 1303.3265. *These authors contributed equally to this work.</div>
<div>[<a data-toggle="collapse" href="#abs_palla2013dependent">Abstract</a>][<a href="#bib_palla2013dependent" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/1303.3265" target="_blank">URL</a>]</div>

<div id="abs_palla2013dependent" class="collapse">
	<b>Abstract</b>: The fundamental aim of clustering algorithms is to partition data points. We consider tasks where the discovered partition is allowed to vary with some covariate such as space or time. One approach would be to use fragmentation-coagulation processes, but these, being Markov processes, are restricted to linear or tree structured covariate spaces. We define a partition-valued process on an arbitrary covariate space using Gaussian processes. We use the process to construct a multitask clustering model which partitions datapoints in a similar way across multiple data sources, and a time series model of network data which allows cluster assignments to vary over time. We describe sampling algorithms for inference and apply our method to defining cancer subtypes based on different types of cellular characteristics, finding regulatory modules from gene expression data from multiple human populations, and discovering time varying community structure in a social network.
</div>
<div id="bib_palla2013dependent" class="collapse">
<b>BibTeX</b>:
<pre>
@article{palla2013dependent,
  author = {Palla*, Konstantina and Knowles*, David A and Ghahramani, Zoubin},
  title = {A dependent partition-valued process for multitask clustering and time evolving network modeling},
  journal = {arXiv},
  year = {2013},
  pages = {1303.3265},
  url = {https://arxiv.org/abs/1303.3265}
}
</pre>
</div>
</li></ol>